### PR TITLE
Add 404

### PR DESCRIPTION
**Pull request by ocefpaf**
@kbailey-noaa and @MathewBiddle this PR removes all the pages in the gh-pages here, sets a 404 page, and a redirect to the new one for the landing page. The redirect is set only to the main landing page, so folks with a direct link to a sub-page will get the 404 instead.

PS: This whole branch, with its history, is available in the https://ioos.github.io/glider-dac gh-pages now.

### CODE REVIEW

File: index.html
Comment (ocefpaf) [reply index.html]: Ooops. Fixed that in the next commit.

Comment (MathewBiddle) [reply index.html]: thx!

**Comment by MathewBiddle** [comment]
LGTM. @kbailey-noaa should have final say before we merge this in.

**Review by kbailey-noaa — CHANGES_REQUESTED** [review]
index.html changes -- the <body> section mentions IOOD CodeLab.

File: index.html
Comment (MathewBiddle) [inline index.html:12]: IOOS Glider DAC

**Comment by ocefpaf** [comment]
> index.html changes -- the section mentions IOOS CodeLab.

Should be OK in the last commit.

**Comment by MathewBiddle** [comment]
LGTM! Thanks @ocefpaf!

**Review by kbailey-noaa — APPROVED** [review]

**Pull request merged**
